includes/fields/class-field-url.php

```diff
@@ -26,15 +26,16 @@ public function render( $field_settings, $form_id ) {
 
             <div class="wpuf-fields">
                 <input
-                    id="<?php echo esc_attr( $field_settings['name'] ) . '_' . esc_attr( $form_id ); ?>"
-                    type="url" class="url <?php echo ' wpuf_'. esc_attr( $field_settings['name'] ).'_'. esc_attr( $form_id ); ?>"
-                    data-duplicate="<?php echo esc_attr( $field_settings['duplicate'] ) ? esc_attr( $field_settings['duplicate'] ) : 'no'; ?>"
-                    data-required="<?php echo esc_attr( $field_settings['required'] ) ?>"
-                    data-type="text"
-                    name="<?php echo esc_attr( $field_settings['name'] ); ?>"
-                    placeholder="<?php echo esc_attr( $field_settings['placeholder'] ); ?>"
-                    value="<?php echo esc_attr( $value ); ?>" size="<?php echo esc_attr( $field_settings['size'] ); ?>"
-                    autocomplete="url"
+                    title = "<?php echo esc_attr__( 'The URL must contain a protocol i.e. http://example.com' ) ?>"

```

Comment:
You'll want to use `'weforms'` as the **domain**. [Search for __(](https://github.com/BoldGrid/weforms/search?q=__) and you'll see what I am referring to.

Reply:
domain has been added to to esc_attr__(